src/EditorFeatures/Core.Wpf/InlineParamNameHints/InlineParamNameHintsTag.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineParamNameHints
+{
+    /// <summary>
+    /// This is the tag which implements the IntraTextAdornmentTag and is meant to
+    /// create the UIElements that get shown in the editor
+    /// </summary>
+    class InlineParamNameHintsTag : IntraTextAdornmentTag
+    {
+        public readonly string TagName;
+
+        /// <summary>
+        /// Creates the UIElement on call
+        /// </summary>
+        /// <param name="text">The name of the parameter associated with the argument</param>
+        public InlineParamNameHintsTag(string text)
+            : base(CreateElement(text), null, (PositionAffinity?)PositionAffinity.Successor)
+        {
+            TagName = text;
+            if (TagName.Length != 0)
+            {
+                TagName += ": ";
+            }
+        }
+
+        private static UIElement CreateElement(string text)
+        {
+            if (text.Length != 0)
+            {
+                text += ": ";
+            }
```

Comment:
```suggestion
            }
            
```

Generally speaking, a blank line is preferred in our codebase between a `}` and teh following statement.